<launch>
  <master auto="start"/>
  <param name="/use_sim_time" value="true"/>
  	<node name="static_tf0" pkg="tf" type="static_transform_publisher" args="2 0 0 0 0 0 /map /base_link 100"/>
   <node name="miscostmaps_node" pkg="miscostmaps" type="miscostmaps_node" output="screen">
      <rosparam file="$(find miscostmaps)/configuration/costmap_common_params.yaml" command="load" ns="global_costmap" />
          <rosparam file="$(find miscostmaps)/configuration/costmap_common_params.yaml" command="load" ns="local_costmap" />
          <rosparam file="$(find miscostmaps)/configuration/local_costmap_params.yaml"  command="load" />
          <rosparam file="$(find miscostmaps)/configuration/global_costmap_params.yaml" command="load" />
        </node>
   <node name="map_server" pkg="map_server" type="map_server" args="$(find miscostmaps)/maps/simple_rooms.yaml" respawn="false"/>
 
  <node pkg="stage_ros" type="stageros" name="stageros" args="$(find mistage1516)/configuracion/mundos/mi-simplerooms.world" respawn="false">
              <param name="base_watchdog_timeout" value="0.2"/>
        </node>
 <node name="fake_localization" pkg="fake_localization" type="fake_localization" respawn="false" />
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find action_lib_scb)/single_robot.rviz" />
</launch>
